Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG NODE_VERSION=18
+ARG BASE_IMAGE_DIGEST=sha256:ea8e360a721d870337fe899c70ea7def62f2a72cf1b6f7beb8a3ccaac8b6049c
```

Comment:
which version is this? could we add a comment (like we do in our .yml's)

Reply:
`node:18-alpine` - and sure, I added a comment!